pkgs/development/tools/casperjs/default.nix

```diff
@@ -28,9 +28,9 @@ in stdenv.mkDerivation rec {
     make test
   '';
 
-
   installPhase = ''
-    mv $PWD $out
+    mkdir $out
```

Comment:
I meant it, so I made https://github.com/NixOS/nixpkgs/pull/15802, feel free to merge one and close the other.